src/api/import.js

```diff
@@ -108,6 +108,15 @@ export default async (providedOptions = {}) => {
     }
   }
 
+  packageJSON.scripts = packageJSON.scripts || {};
+  d('reading current scripts object:', packageJSON.scripts);
+
+  packageJSON.scripts.start = 'electron-forge start';
+  packageJSON.scripts.package = 'electron-forge package';
+  packageJSON.scripts.make = 'electron-forge make';
```

Comment:
Can we at least notify the user if at least one of their scripts got overwritten?